plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchangeManagerFactory.java

```diff
@@ -40,7 +41,7 @@ public String getName()
     @Override
     public ExchangeManager create(Map<String, String> config)
     {
-        requireNonNull(config, "config is null");
+        config = replaceEnvironmentVariables(requireNonNull(config, "config is null"));
```

Comment:
This is done implicitly by Bootstrap.initialize

Reply:
Yes indeed

Reply:
Indeed. My mistake.